src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalSum = days * 40;
+
+  if (days >= 3 && days < 7) {
```

Comment:
if you swap your `if` blocks - you then can omit second check
```suggestion
  if (days >= 3) {
```